gratipay/models/participant/__init__.py

```diff
@@ -341,6 +341,14 @@ def clear_takes(self, cursor):
         """Leave all teams by zeroing all takes.
         """
         for team in self.get_teams():
+            if team.owner == self.username and team.get_take_for(self) == 0:
+
+                # Team owners don't necessarily have takes, and we also don't
+                # hard-require team owners to have a national identity on
+                # file. But we *do* require national identity before we will
+                # set a take--even if setting it to zero.
+
+                continue
```

Comment:
@whit537 Does https://github.com/gratipay/gratipay.com/pull/4480/commits/d723c1ad11d6641bfa270f0a34f31bca3ace502e look good to you?